src/box/tuple.h

```diff
@@ -316,7 +315,15 @@ tuple_data_range(const struct tuple *tuple, uint32_t *p_size)
 }
 
 /**
- * Extract key from tuple by given key definition and return
+ * Sets a key extraction function for the key_def
+ *
+ * @param key_def key_definition
+ *
+ */
+void
+tuple_extract_key_set(struct key_def *key_def);
```

Comment:
Please, follow the existing code - use `def->method = ......_create(def)` instead of `...._set(def)`